src/_tutorials/dart-vm/httpserver.md

```diff
@@ -5,7 +5,6 @@ description: Communicate over the internet
 prevpage:
   url: /tutorials/dart-vm/cmdline
   title: "Dart-VM: Write Command-Line Apps"
-css: ["httpserver.css"]
 ---
 {% capture gh-path -%}
   https://github.com/dart-lang/dart-tutorials-samples/blob/master/httpserver
```

Comment:
> we should probably update that repo

Yes, I'll do that for now (after this PR is accepted), and we'll keep this URL as is (so that readers will be directed to sources w/o `#docregion` tags).

Reply:
Yep, makes sense to keep that repo for the cleaned up version of these files.